src/content/tutorials/connectivity/gateway-api/_index.md

```diff
@@ -0,0 +1,424 @@
+
+The Gateway API consists of three main components available in Giant Swarm:
+
+1. **[Gateway API CRDs](https://github.com/giantswarm/gateway-api-crds-app)**: Core custom resource definitions for Gateway API
+2. **[Envoy Gateway](https://github.com/giantswarm/envoy-gateway-app)**: The gateway implementation based on Envoy Proxy
+3. **[Gateway API Config](https://github.com/giantswarm/gateway-api-config-app)**: Default configuration for quick setup
+
+### Gateway API Resources
+
+- **GatewayClass**: Defines the type of gateway (managed by platform team)
+- **Gateway**: Configures load balancer and listeners (managed by platform team)
+- **HTTPRoute**: Defines HTTP routing rules (managed by application teams)
+- **TCPRoute/UDPRoute**: Defines TCP/UDP routing rules
+- **ReferenceGrant**: Enables cross-namespace references
+
+## Prerequisites
+
+Before setting up Gateway API, ensure you have:
+
+- A Giant Swarm workload cluster
+- `kubectl` configured to access your workload cluster
+- Access to the Giant Swarm platform API for app installation
+- Basic understanding of Kubernetes networking concepts
+
+## Installation
+
+Gateway API support is provided through three apps that work together. You can install them individually or use the Gateway API Bundle for simplified deployment.
+
+### Option 1: Install Gateway API Bundle (Recommended)
+
+The easiest way to get started is using the Gateway API Bundle, which installs all required components:
+
+```bash
+kubectl gs template app \
+  --catalog=giantswarm \
+  --cluster-name=CLUSTER_NAME \
+  --organization=ORGANIZATION \
+  --name=gateway-api-bundle \
+  --target-namespace=envoy-gateway-system \
+  --version=0.3.0 > gateway-api-bundle.yaml
+
+kubectl apply -f gateway-api-bundle.yaml
+```
+
+### Option 2: Install Components Individually
+
+If you need more control over the installation, install each component separately:
+
+#### Step 1: Install Gateway API CRDs
+
+```bash
+kubectl gs template app \
+  --catalog=giantswarm \
+  --cluster-name=CLUSTER_NAME \
+  --organization=ORGANIZATION \
+  --name=gateway-api-crds \
+  --target-namespace=default \
+  --version=1.5.0 > gateway-api-crds.yaml
+
+kubectl apply -f gateway-api-crds.yaml
+```
+
+#### Step 2: Install Envoy Gateway
+
+```bash
+kubectl gs template app \
+  --catalog=giantswarm \
+  --cluster-name=CLUSTER_NAME \
+  --organization=ORGANIZATION \
+  --name=envoy-gateway \
+  --target-namespace=envoy-gateway-system \
+  --version=0.3.0 > envoy-gateway.yaml
+
+kubectl apply -f envoy-gateway.yaml
+```
+
+#### Step 3: Install Gateway API Config (Optional)
+
+For a quick start with default configuration:
+
+```bash
+kubectl gs template app \
+  --catalog=giantswarm \
+  --cluster-name=CLUSTER_NAME \
+  --organization=ORGANIZATION \
+  --name=gateway-api-config \
+  --target-namespace=giantswarm \
+  --version=0.5.1 > gateway-api-config.yaml
+
+kubectl apply -f gateway-api-config.yaml
+```
+
+## Configuration
+
+### Basic Gateway Setup
```

Comment:
this is not needed when using the gateway-api-bundle. it comes with a default Gateway called `giantswarm-default` for the entire cluster.